generator/templates/api.h

```diff
@@ -143,6 +147,8 @@ extern "C" {
 
 #if !defined({{c_prefix}}_SKIP_DECLARATIONS)
 
+{{c_prefix}}_EXPORT {{c_prefix}}Proc {{metadata.namespace}}GetProcAddress(char const * procName);
```

Comment:
Use c_prefix.lower() instead of metadata.namespace because the wgpuGetProcAddress is c function or use as_cMethod(None, Name("GetProcAddress"))

Reply:
fixed.